src/Datasource/Paging/NumericPaginator.php

```diff
@@ -582,7 +582,7 @@ public function validateSort(RepositoryInterface $object, array $options): array
 
         if (
             $options['sort'] === null
-            && count($options['order']) === 1
+            && count($options['order']) >= 1
```

Comment:
That looks logically weird
If the array count is not 0, how can the sort afterwards be chosen?
Is it then just looking at the first Element only?

Reply:
Yes, it wil Look at the First Key element / Key of Order only.

Reply:
Let's say you're configuring pagingParams with MultiOrder.

When you start sending a sort / direction param via url parameter what actually happens is the $options['order'] get's a new array element as first element:

https://github.com/swiffer/cakephp/blob/1a33adfe74ebe2861134c8ad39c2d5e2ec07eb85/src/Datasource/Paging/NumericPaginator.php#L555

That way the result is still ordered by multiple columns but still sort and direction are set (to the first value as set by the query params).